src/index.js

```diff
@@ -80,6 +105,9 @@ export default {
   initialize(sdkKey, callback) {
     AppLovinMAX.initialize(VERSION, sdkKey, callback); // Inject VERSION into native code
   },
+  userSegment,
```

Comment:
Can we declare this right above or under `targetingData`?

Reply:
Moved it to the above targetingData.